woocommerce/woocommerce
# Fix all remaining docs encoding issues

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fixes all remaining unicode encoding issues that block the launch of the docs site. Also bundled are two minor fixes:
* Fix the typo in the localization category name
* Fix the data management category's `README.md` file being incorrectly named



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Validate all modified markdown files and ensure they are still formatted correctly
2. Open the docs site. Open each page and validate that there isn't any other pages that contain encoding issues.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
